CODEOWNERS

```diff
@@ -0,0 +1,46 @@
+# These owners will be the default owners for everything in the repo.
+* @cramforce @dvoytenko
+
+# core javascript owners
+*.js @jridgewell @cramforce @dvoytenko
+
+# core md (docs) owners
+*.md @mrjoro @bpaduch
+
+# top level file owners
+caches.json @cramforce @dvoytenko
+gulpfile.js @ampproject/runtime
+.babelrc @erwinmombay @rsimha-amp
+.eslintrc @erwinmombay @rsimha-amp
+.gitignore @erwinmombay @rsimha-amp
+.travis.yml @erwinmombay @rsimha-amp
+yarn.lock @erwinmombay @rsimha-amp
+
+# folder owners
+3p/* @ampproject/runtime
+src/* @ampproject/runtime
+ads/* @lannka @zhouyx
+css/* @ampproject/runtime
+examples/* @ampproject/runtime
+extensions/* @ampproject/runtime
+builtins/* @ampproject/runtime
+screenshots/* @ampproject/runtime
+contributing/* @mrjoro @bpaduch
+spec/* @mrjoro @bpaduch
+testing/* @ampproject/runtime
+build-system/* @erwinmombay @cramforce
```

Comment:
Does this system allow the addition of separate owners for some of the files / sub folders under this, while retaining you guys as top level folder owners? (It might make sense to add me as owner of some of the gulp tasks / test infra.)